src/index.js

```diff
@@ -41,7 +41,7 @@ App.get( `${ Settings.endpoint }*`, async ( request, response ) => {
 	);
 
 	// Notify Slack!
-	SendSlackMessage( `Hello there!` );
+	SendSlackMessage( `Karma-Karma-Karma-Chameleon! \nGenerating template for ${request.path} using ${ColorMapToString( request.query )}` );
```

Comment:
Beautiful message 💯, is it possible to make it send it out like furnace does. Right now this just logs a long message into slack. It would be nice to get new lines etc.